package.json

```diff
@@ -1,16 +1,13 @@
 {
-  "name": "@greetz/node-metrics",
-  "version": "1.0.3",
+  "name": "@greetznl/node-metrics",
+  "version": "1.0.4",
   "description": "Client for gathering metricizes from nodejs application and exposing them on endpoints",
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 0",
     "lint": "eslint lib"
   },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/GreetzNL/metrics-client.git"
-  },
+  "repository": "git@github.com:GreetzNL/node-metrics.git",
```

Comment:
is this just the shortened version of "repository" property?

Reply:
Also, don't you think we need to also specify that this is a Github package, because I think by not mentioning it, it automatically is being classified as npm package